docker/go-1.10/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Go cross compiler (xgo): Go 1.10
+# Copyright (c) 2017 Péter Szilágyi. All rights reserved.
+#
+# Released under the MIT license.
+
+FROM karalabe/xgo-base
+
+MAINTAINER Péter Szilágyi <peterke@gmail.com>
+
+# Configure the root Go distribution and bootstrap based on it
+ENV GO_VERSION 110
```

Comment:
This version number will blow up, because it would cause 1.10 (`110`) to be < 1.5 (`150`). Can't say what the nicest solution would be. Perhaps we could go with `1100`?

Reply:
I built it locally with GO_VERSION set to 1100. Its not pretty but it works.

Reply:
Thanks, @karalabe and @CrawX Pushed changes, please take a look now.